src/PowerBIEmbedded/PowerBIEmbedded.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27703.2042
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.32602.291
```

Comment:
Why are we updating Visual Studio version in this PR?

Reply:
These changes are results of rebuilding the solution.